### PR TITLE
feat: add translations workflow for GitHub releases

### PR DESCRIPTION
**Pull request by kbdharun**

### CODE REVIEW

**Review by SethFalco — CHANGES_REQUESTED** [review]
Take a peek at the CONTRIBUTING guide:

https://github.com/tldr-pages/tldr-translation-pairs-gen/blob/main/CONTRIBUTING.md#backward-compatibility

It's cool if we create new assets, but preferably, we shouldn't change how we package the original artifacts.

TL;DR: Makes sense, but can we _add_ the new artifacts instead of _replace_ the existing artifact.

**Comment by kbdharun** [comment]
> It's cool if we create new assets, but preferably, we shouldn't change how we package the original artifacts.

 Maybe, I will separate this workflow and we can continue to have the existing one that way the GitHub page would be present and we can upload the ZIPs to releases for ease of downloading and redistributing in places like Kaggle.

File: .github/workflows/translations-release.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    - name: Fetch sources
+      run: |
+        wget https://github.com/tldr-pages/tldr/archive/refs/heads/main.zip --output-document /tmp/main.zip
+        unzip /tmp/main.zip -d /tmp
+
+    - name: Generate CSV dataset
+      run: |
+        npm run tldr-translation-pairs-gen -- --source /tmp/tldr-main --output tldr-translation-pairs-dataset --format csv
+        zip tldr-translations-dataset.zip tldr-translation-pairs-dataset/*
```
Comment (SethFalco) [inline .github/workflows/translations-release.yml:45]:
```suggestion
        npm run tldr-translation-pairs-gen -- --source /tmp/tldr-main --output tldr-translation-pairs-dataset-csv --format csv
        zip tldr-translations-dataset-csv.zip tldr-translation-pairs-dataset-csv/*
```

**Review by SethFalco — COMMENTED** [review]
LGTM!

Fine with this overall, but perhaps the CSV dataset should have CSV in the name similarly to the others for consistency?

That way when someone reviews the releases, users just know it's CSV rather than having to guess what the unspecified format contains.

**Review by SethFalco — APPROVED** [review]

**Pull request merged**